server/rpc/server_test.go

```diff
@@ -735,7 +735,7 @@ func TestAdminRPCServerBackend(t *testing.T) {
 				Password: invalidSlugName,
 			},
 		)
-		assert.Equal(t, codes.Unauthenticated, status.Convert(err).Code())
+		assert.Error(t, err)
```

Comment:
Is there any reason to remove the status code check?

Reply:
It was for testing purpose, it should not be included.

I have reverted the code.